main.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'app'
+def main
+  print('Welcome to School Library App!')
+  app = App.new
+  app.run
+end
+main
```

Comment:
- [ ] Good job creating the UI for the school library 👏 . The UI allows creating book and index and to fetch data however when I tried to create a rental , It is throwing the following exception. Kindly fix that 😄 

![Capture](https://github.com/MasumaJaffery/oop-School-Library/assets/80458199/5bcf430d-2420-4878-8fcd-58e9b7f6c280)